source/backend/apimodels/Models/Concepts/Property/PropertyViewModel.cs

```diff
@@ -0,0 +1,101 @@
+using System;
+
+namespace Pims.Api.Models.Concepts.Property
+{
+    /// <summary>
+    /// PropertyViewModel class, provides a model to represent the property view.
+    /// </summary>
+    public class PropertyViewModel
+    {
+        #region Properties
+
+        public long Id { get; set; }
+
+        public int? Pid { get; set; }
+
+        public string PidPadded { get; set; }
+
+        public int? Pin { get; set; }
+
+        public string PropertyTypeCode { get; set; }
+
+        public string PropertyStatusTypeCode { get; set; }
+
+        public string PropertyDataSourceTypeCode { get; set; }
+
+        public DateOnly PropertyDataSourceEffectiveDate { get; set; }
+
+        public string PropertyClassificationTypeCode { get; set; }
+
+        public string PropertyTenureTypeCode { get; set; }
+
+        public string StreetAddress1 { get; set; }
+
+        public string StreetAddress2 { get; set; }
+
+        public string StreetAddress3 { get; set; }
+
+        public string MunicipalityName { get; set; }
+
+        public string PostalCode { get; set; }
+
+        public string ProvinceStateCode { get; set; }
+
+        public string ProvinceName { get; set; }
+
+        public string CountryCode { get; set; }
+
+        public string CountryName { get; set; }
+
+        public string Name { get; set; }
+
+        public string Description { get; set; }
+
+        public long? AddressId { get; set; }
+
+        public short RegionCode { get; set; }
+
+        public short DistrictCode { get; set; }
+
+        //public Geometry Geometry { get; set; }
```

Comment:
If this isn't necessary please remove it. The next developer won't know why this is here.